setup-runner.sh

```
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# GitHub Actions Self-Hosted Runner Setup Script
+# This script sets up a self-hosted GitHub Actions runner for this repository
+
+set -e
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)